src/ConstructiveSolidGeometry/SurfacePrimitives/Polygon.jl

```diff
@@ -1,5 +1,5 @@
 
-struct Polygon{N,T} <: AbstractPlane{T}
+struct Polygon{N,T} <: AbstractPlanarSurfacePrimitive{T}
     points::SVector{N, CartesianPoint{T}} 
```

Comment:
Is it intended that `Polygon` has no `origin` and `rotation`? How is this dealt with when a `HexagonalPrism` is translated or rotated?

Reply:
Yes, it is not needed. The points of a polygon are in always in the global coordinate system. 
It also does not have a real orientation.